entrypoint.sh

```diff
@@ -42,7 +52,27 @@ if [ "$INPUT_PDF" = "true" ]; then
   pandoc "${PANDOC_ARGS[@]}" -t latex --template=/cabforum/templates/guideline.latex -o "${BASE_FILE}.tex" "${1}"
   TEXINPUTS="${TEXINPUTS}:/cabforum/" pandoc "${PANDOC_PDF_ARGS[@]}"
   set +x
+  echo "::set-output name=pdf_file::${BASE_FILE}.pdf"
   echo "::endgroup::"
+
+  if [ ! -z "${DIFF_FILE}" ]; then
+    echo "::group::Generating diff"
+    TMP_DIR=$(mktemp -d)
+    OUT_DIFF_TEX=$(basename "${DIFF_FILE}")
+    OUT_DIFF_TEX="${TMP_DIR}/${OUT_DIFF_TEX%.*}"
```

Comment:
You could skip the `%.*` here by passing `-s .md` to basename on the line above.